v2/oauth/reuse-website-login.mdx

```diff
@@ -0,0 +1,273 @@
+The recipe will expose the endpoints needed for enabling the [**OAuth 2.0**](/docs/oauth/introduction#oauth-20) flow.
+
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+Update the `supertokens.init` call to include the `OAuth2Provider` recipe.
+
+1. Add the import statement for the recipe.
+
+```typescript 
+import OAuth2Provider from "supertokens-node/recipe/oauth2provider";
+```
+
+2. Update the list of recipes with the new initialization step.
+
+```tsx
+OAuth2Provider.init()
+```
+
+</TabItem>
+
+<TabItem value="go">
+
+:::caution
+
+At the moment we do not have support creating OAuth2 providers in the Go SDK.
+
+:::
+
+</TabItem>
+
+<TabItem value="python">
+
+:::caution
+
+At the moment we do not have support creating OAuth2 providers in the Python SDK.
+
+:::
+
+</TabItem>
+
+</BackendSDKTabs>
+
+
+###  Configure the Authorization Service Frontend 
+
+
+Initialize the `OAuth2Provider` recipe on the frontend of your **Authorization Service**. 
+
+:::info
+
+If you want to use your own custom UI check our [separate guide](/docs/oauth/customizations/custom-ui) that explains
+all the steps that you have to take into account.
+
+:::
+
+<OAuthFrontendTabs>
+
+
+<TabItem value="react">
+
+Update the `SuperTokensConfig` variable to include the `OAuth2Provider` recipe.
+
+1. Add the import statement for the recipe.
+
+```typescript 
+import OAuth2Provider from "supertokens-auth-react/recipe/oauth2provider";
+```
+
+2. Update the list of recipes to also include the new initialization.
+
+```tsx
+OAuth2Provider.init()
+```
+    
+</TabItem>
+
+
+<TabItem value="angular">
+
+Update the `AuthComponent` so that it also includes the `OAuth2Provider` recipe.
+You will have to add a new item in the `recipeList` array, inside the `supertokensUIInit` call.
+
+```typescript
+
+supertokensUIOAuth2Provider.init(),
+
+```
+
+</TabItem>
+
+<TabItem value="vue">
+
+Update the `AuthView` component so that it also includes the `OAuth2Provider` recipe.
+You will have to add a new item in the `recipeList` array, inside the `supertokensUIInit` call.
+
+```typescript
+
+supertokensUIOAuth2Provider.init(),
+
+```
+    
+</TabItem>
+    
+</OAuthFrontendTabs>
+
+
+
+## 4. Set Up Session Handling in Each Application
+
+In each of your individual `applications` you will have to setup up logic for handling the **OAuth 2.0** authentication flow.
+We recommend using a generic **OICD** or **OAuth** library in order to do this.
+
+<OAuthMobileTabs>
+
+<TabItem value="react-native">
+
+You can use the [react-native-app-auth](https://commerce.nearform.com/open-source/react-native-app-auth/) library. 
+Just follow [the instructions](https://commerce.nearform.com/open-source/react-native-app-auth/docs/usage/config) setup your application. 
+The configuration parameters can be determined based on the response that we received on **step 2**, when we created the **OAuth2 Client**.
+
+- `issuer` corresponds to the endpoint of the **Authorization Service** `^{form_apiDomain}`
```

Comment:
i think the issuer that we use is apidomain + apibasepath. Please confirm with @porcellus